persistent/tests/test_persistence.py

```diff
@@ -586,6 +666,9 @@ def test___getattribute___p__names(self):
         for name in NAMES:
             getattr(inst, name)
         self._checkMRU(jar, [])
+        # _p_mtime is special, it activates the object
+        getattr(inst, '_p_mtime')
+        self._checkMRU(jar, [OID])
```

Comment:
I really don't understand why it should be activated.  Is there a real usecase, or are we just blindly following a bug in the C implementation?